templates/docs/patterns/labels.md

```diff
@@ -10,12 +10,6 @@ context:
 
 Labels are static elements which you can apply to signify status, tags or any other information you find useful.
 
-<div class="p-notification--information">
```

Comment:
Hi @bartaz! All looks good, just wanted to make sure this deletion was intentional?

Reply:
@mtruj013 Yes, this notification was not relevant anymore (initially this component was meant for Vanilla status only, but now is used elsewhere as well).